src/BenchmarksDriver/Program.cs

```diff
@@ -363,6 +370,7 @@ public static int Main(string[] args)
                         [Duration] [int] NOT NULL,
                         [PipelineDepth] [int] NULL,
                         [Headers] [nvarchar](max) NULL,
+                        [Method] [nvarchar](max) NOT NULL,
```

Comment:
Nit: I would move this above `Headers`, since the method comes before headers in an HTTP request.